### PR TITLE
Adds contributing guidelines and changelog

### PR DESCRIPTION
**Pull request by ben-dalton**
@johnallen3d 

This PR adds CONTRIBUTING.md as well as CHANGELOG.md to help encourage consistency in contributions and a record of changes.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Next Release
+
+- Your contribution here.
+- [#35](https://github.com/technekes/cast-ui/pull/35): Adds pagination component - [@ben-dalton](https://github.com/ben-dalton).
+
+# 0.1.8 (02/07/2019)
```
Comment (johnallen3d) [inline CHANGELOG.md]: Prefer ISO-8601 dates?

iso-8601

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Next Release
+
```
Comment (johnallen3d) [inline CHANGELOG.md]: Consider mentioning [Keep a Changelog](http://keepachangelog.com) here? Perhaps something like this...

> The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).

File: CHANGELOG.md (hunk 1 above)
Comment (ben-dalton) [reply CHANGELOG.md]: Haha. I am partial to some of the latter options.

File: CHANGELOG.md (hunk 2 above)
Comment (ben-dalton) [reply CHANGELOG.md]: great idea!

**Pull request merged**